docker/Dockerfile

```diff
@@ -10,64 +10,48 @@ ENV TERM linux
 ENV BASH_ENV ~/.bashrc
 SHELL ["/bin/bash", "-c"]
 
-# comment out the line in the default .bashrc that does an early exit if not in
-# interactive mode. It should be noted that the -i option of docker run does
-# not launch the shell in interactive mode. CCheck Dockerfile.3 for a demo.
-RUN sed -e '/[\ -z \"\$PS1\"\ ]\ &&\ return/ s/^#*/#/' -i /root/.bashrc
-
 # General installs in the docker
-RUN apt-get -y update && apt-get -y upgrade
-
-RUN apt-get -y install \
-    apt-utils \
-    build-essential \
-    software-properties-common
-
-RUN apt-get install -y \
-    curl \
-	git \
-	screen \
-	ufw \
-    vim \
-	wget
-
-# Install miniconda and add to PATH
-RUN wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
-RUN sh Miniconda3-latest-Linux-x86_64.sh -b
-ENV PATH /root/miniconda3/bin:$PATH
-
-# expose ports for flask app - both server and backoffice
-EXPOSE 8888 8890
+RUN apt-get -y update \
+    && apt-get install -y git \
+       wget \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
can also run a `apt-get clean -y` here.

Reply:
Indeed. added on my side (although in this case there's no visible change on the image size)